Sources/Public/Animation/LottieView.swift

```diff
@@ -67,17 +95,27 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
   /// Creates a `LottieView` that asynchronously loads and displays the given `LottieAnimationSource`.
   /// The `loadAnimation` closure is called exactly once in `onAppear`.
   /// While the animation is loading, the `placeholder` view is shown in place of the `LottieAnimationView`.
+  /// - Parameters:
+  ///   - loadAnimationTrigger: `Binding` that  triggers a new call to `loadAnimation` when a new value is propagated
+  ///   - loadAnimation: Closure that provides an animation asynchronously
   public init(
+    loadAnimationTrigger: Binding<AnyHashable> = .constant(LoadReference.constantSource),
     loadAnimation: @escaping () async throws -> LottieAnimationSource?,
     @ViewBuilder placeholder: @escaping () -> Placeholder)
   {
     self.loadAnimation = loadAnimation
     self.placeholder = placeholder
+    self.loadAnimationTrigger = loadAnimationTrigger
     _animationSource = State(initialValue: nil)
   }
 
   // MARK: Public
 
+  /// Token used when no `loadAnimationTrigger` is provided
+  public enum LoadReference {
```

Comment:
Instead of using `loadAnimationTrigger = .constant(LoadReference.constantSource)` to mean "call this only once", could we make it optional and use either `loadAnimationTrigger = nil` or `loadAnimationTrigger = .constant(nil)`? 

It would be nice to not add a new public value just for this functionality